src/main/java/seedu/smarthomebot/Main.java

```diff
@@ -0,0 +1,80 @@
+package seedu.smarthomebot;
+
+import seedu.smarthomebot.logic.commands.Command;
+import seedu.smarthomebot.logic.commands.CommandResult;
+import seedu.smarthomebot.logic.commands.ExitCommand;
+import seedu.smarthomebot.commons.Messages;
+import seedu.smarthomebot.logic.parser.Parser;
+import seedu.smarthomebot.data.appliance.ApplianceList;
+import seedu.smarthomebot.data.location.LocationList;
+import seedu.smarthomebot.storage.ReadStorageFile;
+import seedu.smarthomebot.storage.WriteStorageFile;
+import seedu.smarthomebot.ui.TextUi;
+
+/**
+ * Entry point of the SmartHome application.
+ * Initializes the application and starts the interaction with the user.
+ */
+
+public class Main {
+
+    private TextUi ui = new TextUi();
```

Comment:
In your class diagram, there is no need to put ui as an attribute and also use a line to denote composition. Just the line would do
![Uploading Screenshot 2020-10-29 at 16.13.30.png…]()